sdk/textanalytics/Azure.AI.TextAnalytics/src/ServiceClients/LanguageServiceClient.cs

```diff
@@ -16,87 +20,2193 @@ namespace Azure.AI.TextAnalytics.ServiceClients
     ///
     internal class LanguageServiceClient : ServiceClient
```

Comment:
This is where we'll need to shift effort for integrating the Language REST API.

//cc: @nisha-bhatia